GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -61,6 +61,24 @@ protected override List<MapEntityMetadata> Parse()
                 mapObjects.Add(modelName, thisNode);
             }
 
+            // fetch interactID and recipeID relation from xml (can be expanded to parse other xml info)
+            Dictionary<int, int> interactRecipeMap = new Dictionary<int, int>();
+            foreach (PackFileEntry entry in Resources.XmlFiles
+                .Where(entry => Regex.Match(entry.Name, "table/interactobject_mastery").Success)
+                .OrderBy(entry => entry.Name))
```

Comment:
does it need to be ordered?

Reply:
You are right no need to be ordered since it's only a single file